### PR TITLE
Use test fixture for a test.

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

File: test/NamedId.cpp [hunk 1]
```diff
@@ -24,16 +24,17 @@
...
+#include "UnitSPIRV.h"
+
 namespace {
 
-TEST(NamedId, Default) {
-  spv_context context = spvContextCreate();
-  const char* spirv = R"(
+using spvtest::TextToBinaryTest;
+
+TEST_F(TextToBinaryTest, Default) {
```
Comment (dmircevski) [inline test/NamedId.cpp]: Why change a specific test name into a generic one?

Comment (antiagainst) [inline test/NamedId.cpp]: I can create an alias for TextToBinaryTest.

Comment (antiagainst) [inline test/NamedId.cpp]: Done.

**Comment by dmircevski** [comment]
+2

**Pull request merged**